compiler/cpp/src/thrift/generate/t_rs_generator.cc

```diff
@@ -350,6 +350,7 @@ class t_rs_generator : public t_generator {
   void render_sync_handler_failed_default_exception_branch(t_function *tfunc);
   void render_sync_handler_send_exception_response(t_function *tfunc, const string &err_var);
   void render_service_call_structs(t_service* tservice);
+  void render_args_struct(t_function* tfunc);
```

Comment:
Perhaps rename to `render_service_args_struct`?

Reply:
Or `render_service_call_args_struct`?

Reply:
`render_service_call_args_struct` sounds good to me! I'll also change `render_result_value_struct` into `render_service_call_result_value_struct` to stay consistent 🙂